windows/src/desktop/kmshell/main/OnlineUpdateCheck.pas

```diff
@@ -559,11 +545,12 @@ function TOnlineUpdateCheck.DoRun: TOnlineUpdateCheckResult;
     with THTTPUploader.Create(nil) do
     try
       ShowUI := not FSilent;
-      Fields.Add('Version', ansistring(FCurrentVersion));
+      Fields.Add('version', ansistring(CKeymanVersionInfo.Version));
+      Fields.Add('tier', ansistring(CKeymanVersionInfo.Tier));
       for i := 0 to kmcom.Packages.Count - 1 do
       begin
         pkg := kmcom.Packages[i];
-        Fields.Add(ansistring('Package_'+pkg.ID), ansistring(pkg.Version));
+        Fields.Add(ansistring('package_'+pkg.ID), ansistring(pkg.Version));
```

Comment:
I imagine not, since it looks to be a deliberate change, but just to be sure:

- This isn't going to cause case-sensitivity issues when updating from prior versions, is it?
- Alternatively, will the change that motivated this cause issues for previous versions?

Reply:
This is using the new 14.0 endpoint, so doesn't impact earlier versions.